hibernate-core/src/test/java/org/hibernate/test/annotations/duplicatedgenerator/DuplicateTest.java

```diff
@@ -31,13 +32,14 @@ public void testDuplicateEntityName() throws Exception {
 		try {
 			cfg.addAnnotatedClass( Flight.class );
 			cfg.addAnnotatedClass( org.hibernate.test.annotations.Flight.class );
+			cfg.addAnnotatedClass( org.hibernate.test.annotations.Company.class );
```

Comment:
Why add this?

Reply:
Because the test was wrong. The Flight entity has an association to Company.